### DISCUSSION TITLE
Switching between touchscreen and 12864 LCD emulation modes with RRF

### DISCUSSION THREAD

**Discussion by Wombat37** [bigtreetech/BIGTREETECH-TouchScreenFirmware#2213] [General]
When I first bought my BTT RRF-E3-v1.1 and TFT35-E3-v3.0, I understood that I could use either the touchscreen mode or the 12864 emulation mode - cables are provided in the kit for both. Holding down the rotary encoder knob gives me the option to switch between them.

I'm running RepRap firmware and I quickly realized that the 12864 wasn't supported. Now I see that RepRap does support the 12864.

So can I now switch between the two modes and, if so, what do I have to put into the config.ini file to allow this to happen? 

I really like the touchscreen mode so I don't know if I have a need for 12864 emulation but it would be good to know what's possible.

Has anybody done this or have information on it?

**Comment by oldman4U** [comment]
@bttguy

Can you please help. 

Thank you

**Comment by radek8** [comment]
https://teamgloomy.github.io/skr_rrf_e3_screen_12864.html

**Comment by Wombat37** [comment]
Yes - I had seen that. I can see how to configure one mode or the other but not how to switch between them.

My question relates to how is the display mode is switched between between touch screen and 12864 emulation mode?

Do I put configuration code for both modes in the board.txt and config.g fles and connect both cables to the TFT? Would the TFT then be able to switch modes by holding down the encoder knob?

Sorry for not making the question clearer.

**Comment by radek8** [comment]
Yes
If you configure both modes and connect cables for both modes, you can switch between them.
Do not use an encoder to switch. Use your finger on the display to switch. Do not use clicking on the encoder in touch mode, otherwise you may have a problem.

**Comment by Wombat37** [comment]
Thank you for your advice - it is much appreciated. 

I'll try this out.
